### PR TITLE
[API] POST ADMIN (Middleware + Controller)

### PR DESCRIPTION
**Pull request by Debatreya**
# Middleware:
- Takes {userId, email} from req.cookies. // ID is not in SCHEMA -> Resolved
- Find user from User model
- res.404 if User doesnot exist
- res.401 unauthorized if User is not admin
- next()

# Post Admin
- Not sure about if anything to be done with req.body // Commented
- Takes out all users -> Removed 
- Takes out Blogs that are published (review: Approved}
- Takes out Blogs that have update review (revied : update)
- Takes out Blogs that need review (review: pending)
- return everything in res.

@Himasnhu-AT Please review this PR.

### CODE REVIEW

File: server/src/router/post-admin.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    try {
+        // Published Blogs (where review: approved)
+        const published_blogs = await Blog.find({ review: "approved" });
+        // Updated Reviews (where review: update)
+        const updated_review = await Blog.find({ review: "update" });
+        // Pending Reviews (where review: pending)
+        const review = await Blog.find({ review: "pending" });
+        // All Users
+        const all_users = await User.find();
```
Comment (Himasnhu-AT) [inline server/src/router/post-admin.js]: No need to return all users.

**Review by Himasnhu-AT — CHANGES_REQUESTED** [review]
No need to return allUsers in /admin api call, as we don't require it...

@Debatreya @nototiger0227

**Comment by Debatreya** [comment]
@Himasnhu-AT check this PR.

@nototiger0227 this is the Final PR. Good Work, Thanks For Showing up.

**Review by Himasnhu-AT — APPROVED** [review]

**Comment by Himasnhu-AT** [comment]
Great job @Debatreya @nototiger0227 on closing this issue :)

**Comment by Debatreya** [comment]
@Himasnhu-AT 
yeh tests konse fail ho rhe hain?

**Comment by Himasnhu-AT** [comment]
Static analysis 

Basically rate limiting and credential leaks. Resolved later

**Pull request merged**